### ISSUE TITLE
Add keyboard shortcut to toggle visibility of annotations (highlights) on host page

### ISSUE THREAD

**Issue by lyzadanger** [hypothesis/client#4740]
> On heavily annotated documents, the start and end annotation messages can make the reading experience very strange and unusable. There is a way to turn off annotations that solves this problem...however, that function should utilize a keyboard command as well so users can enable it on the fly.

Notes from initial tech discussion:

* We need to orchestrate this such as to not conflict with system keyboard shortcuts or shortcuts in play on the current host page

**Comment by robertknight** [comment]
Discussion of some shortcut options: https://hypothes-is.slack.com/archives/C1M8NH76X/p1662389998993489
